circle.yml

```diff
@@ -65,3 +82,12 @@ test:
             background: true
         - sleep 5
         - mocha-phantomjs http://localhost:5000/base/test/index.html
+
+        # Docker
+        - docker run -d --name db postgresql
+        - docker run -d --name solr solr
+        - docker run -d --name redis redis:latest
+        - docker run -d --name ckan -p 5001:5000 --link db:db --link redis:redis --link solr:solr ckan
+        - sleep 10
+        - curl --retry 10 --retry-delay 5 -v http://localhost:5001
+
```

Comment:
You've added about 2 minutes to the all CI containers. Would you add your docker build and test code to https://github.com/ckan/ckan/blob/master/.circleci-matrix.yml so that it only runs in one container instead of all 4?